### ISSUE TITLE
Help CDT setup cross region redundancy without firing up servers

### ISSUE THREAD

**Issue by aaronhans** [cagov/covid19#955]
Continue working with CDT team to implement another instance of covid19.ca.gov that does not have any server components. We already have a version of this we setup at staging.covid19.ca.gov using Azure static web app service but since that is still in preview status we aren't going to use it in production. We proposed blob storage configured as a website instead and that seemed acceptable to CDT. Will work with them to make sure we find a solution that doesn't have servers.

**Comment by aaronhans** [comment]
We failed here. There was too much pressure on CDT to checkoff the cross region redundancy box so they fired up more Windows servers.

We did succeed in getting on the same page as far as the end goal though. The steps to reach the end goal involve CDT decoupling their CDN from the cloud provider in their reach for cross cloud provider redundancy (meaning Azure+AWS+GCP). We will be helping create the correct architecture pattern ourselves on AWS in #957. 

CDT heard from Microsoft that their best in class static web app service will not be promoted out of preview anytime soon so will not receive the required uptime contract to be used in production. The alternative to that on Azure is basic blob storage configured as a website which provides the same basic functionality. It is missing some cool features like automatically spinning up browseable environments on opened PRs.

This ticket should be closed. We made friends along the way but failed to headoff increasing infrastructure complexity.
